public/js/sign-up.js

```diff
@@ -0,0 +1,66 @@
+const username = document.getElementById("user");
```

Comment:
check this [validation topic](https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/Form_validation#Validating_forms_using_JavaScript) for later projects

Reply:
okay , thanks